chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,79 @@
+or argument you really want to use.
+
+### Enable Intellisense for jina yaml configs in Pycharm:
+
+- First open Pycharm, click on the Menu and select Preference. 
+- In the left panel, search for JSON schema in the search box and select JSON schema mappings.
+- Next, in the right panel click + icon and add a new Schema.
+- Name: Name as per choice (for eg. jina) 
+- In the Schema file or URL write `api.jina.ai/schemas/latest.json` ( this ensures that your schema is always up to date with the new release).
+- You can also bind it to the specific Jina version or pre-release version.
+- In the Schema Version, select JSON Schema version 7.
+- Finally, add to File path pattern mapping: *.jaml to associate any files that end with JAML or Jina yaml extensions to the schema.
+- Click OK to apply the change.
+
+![ide-speed-up](intelli.png "Enabling Intellisense")
+
+### For Visual Studio Code developers:
+
+- First install the YAML extension from the RedHat.
+- Click on the sidebar and select Extension.
+- Search for YAML and then select *YAML support* from RedHat and then install it.
+- Open your IDE-wise `settings.json`, create a new entry:
+```
+“yaml.schemas”: {
+“https://api.jina.ai/schemas/latest.json”: [“/*.jina.yml, /*.jaml]
+}
+```
+- Finally save your `settings.json` to apply the change.
+
+![ide-speed-up](vse.png "VSCode Autocomplete")
+
+
+Create a new hello.jina.yml file and the IDE successfully marks it as Jina file type
+If it doesn't, you may want to manually select the schema you just created.
+Now type Jtype in the first line and you will see the IDE suggests you with flow or a list of executors that jina contains.
+
+![ide-speed-up](pyc.png "Pycharm Autocomplete")
+
+*jtype* is a synonym for the bond mark. We recommend you to use j-type over the bond mark as
+it gives a cross-platform compatible yaml files.
+Now, the IDE immediately marks it as yellow as it is not a valid jina config file.
+Hover your mouse on it and it complains you haven't defined the required field version and parts.
+It let's it complete them as you write urlc's autocomplete kicks in to help you fill in the default
+values. You can hover your mouse on the field to see the help text, which is consistent with the
+documentation.
+You will also see the hint is based on the context: for example, when writing parts, it
+suggests all arguments that are accepted by the part.
+When you write something that is unrecognizable by the schema, the ID will mark it as yellow
+immediately. The intellisense becomes super helpful when you write an executor level yaml
+file.
+Let's create a new YAML file for configuring Numpy indexer.
+You can see the IDE will only keep the arguments that are accepted by that executor.
+It also works on the nested level when you define request and it shows only drivers
+and when you define the drivers it shows relevant arguments that are accepted by
+that driver.
```

Comment:
TBH, it is quite daunting in the current format. You might want to rephrase this part and split them into several paragraphs.